dropwizard-jackson/src/main/java/io/dropwizard/jackson/AnnotationSensitivePropertyNamingStrategy.java

```diff
@@ -9,18 +10,13 @@
 /**
  * A {@link PropertyNamingStrategy} implementation which, if the declaring class of a property is
  * annotated with {@link JsonSnakeCase}, uses a
- * {@link com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy}, and uses
+ * {@link com.fasterxml.jackson.databind.PropertyNamingStrategies.SnakeCaseStrategy}, and uses
  * the default {@link PropertyNamingStrategy} otherwise.
  */
 public class AnnotationSensitivePropertyNamingStrategy extends PropertyNamingStrategy {
     private static final long serialVersionUID = -1372862028366311230L;
 
-    private final SnakeCaseStrategy snakeCase;
-
-    public AnnotationSensitivePropertyNamingStrategy() {
-        super();
```

Comment:
Do we need to keep the constructor to call super()?

Reply:
The default constructor does that.
> The compiler automatically provides a no-argument, default constructor for any class without constructors. This default constructor will call the no-argument constructor of the superclass.

https://docs.oracle.com/javase/tutorial/java/javaOO/constructors.html

Reply:
Also there is no explicit constructor on the superclass (at least in the current version) so `super()` just ended up calling the default constructor AFAIK.